src/alarms/notifications/App.js

```diff
@@ -26,29 +26,30 @@ class App extends Component {
       this
     );
     this.loadAlarmsOnPage = this.loadAlarmsOnPage.bind(this);
+    this.activateAlarm = this.activateAlarm.bind(this);
+    this.deActivateAlarm = this.deActivateAlarm.bind(this);
+    this.removeAlarm = this.removeAlarm.bind(this);
   }
   componentDidMount() {
     const { page } = this.state;
     this.loadAlarmsOnPage(page);
   }
-  // componentWillReceiveProps(newProps) {
-  //   console.log("componentWillReceiveProps notifications app");
-  //   const { page, hasFiredFetch } = this.state;
-  //   const { isFetchingOrganisations, selectedOrganisation } = newProps;
-  //   console.log(
-  //     "isFetchingOrganisations",
-  //     isFetchingOrganisations,
-  //     selectedOrganisation
-  //   );
-  //   if ( !hasFiredFetch && selectedOrganisation) {
-  //     this.loadAlarmsOnPage(page, newProps);
-  //   }
-  // }
 
   loadAlarmsOnPage(page, newProps) {
+    this.setState({
+      isFetchingTimeseriesAlarms: true,
+      isFetchingRasterAlarms: true,
+      total: 0,
+      alarms: []
+    });
+
```

Comment:
Alarmen worden nu ook opnieuw gefetched nadat ze verwijdert zijn of zijn gewisseld naar actief/passief.
Dit maakt het noodzakelijk om de state te legen voordat ze opnieuw gefetched worden. Dit wordt mede veroorzaakt dor het feit dat alarmen van 2 sources komen. De promise met alarmen mag dus niet simpelweg alle bestaande alarmen overschrijven, want hier kunnen de alarmen van de vorige promise al inzitten.